ocaml/src/proxy_client.ml

```diff
@@ -226,8 +230,26 @@ let make_client ip port transport =
     ip port transport
   >>= fun (nfd, closer) ->
   Lwt.catch
-    (fun () -> _prologue nfd Protocol.magic Protocol.version >>= fun () ->
-               Lwt.return (new proxy_client nfd, closer))
+    (fun () ->
+      _prologue nfd Protocol.magic Protocol.version >>= fun () ->
+      let session = ProxySession.make () in
+      let client = new proxy_client nfd session in
+      let manifest_ser = 2 in
+      let kvs = [("manifest_ser", Some (serialize Llio.int8_to manifest_ser))]
+      in
+      client # update_session kvs >>= fun processed ->
```

Comment:
This might fail with UnknownOperation on older proxies, so for backwards compatibility that error should be caught and ignored